docs/docs/path_algorithm.md

```diff
@@ -0,0 +1,406 @@
+---
+title: "Path algorithms"
+linkTitle: "Path algorithms"
+description: "Learn how to use algo.SPpaths and algo.SSpaths to solve queries"
+weight: 7
+---
+
+RedisGraph 2.10 introduced two new path-finding algorithms, or more accurately, minimal-weight, optionally bounded-cost, and optionally bounded-length path-finding algorithms, `algo.SPpaths` and `algo.SSpaths`.
+
+`algo.SPpaths` and `algo.SSpaths` can solve a wide range of real-world problems, where minimal-weight paths need to be found. algo.SPpaths finds paths between a given pair of nodes, while algo.SSpaths finds paths from a given source node. Weight can represent time, distance, price, or any other measurement. A bound can be set on another property (e.g., finding a minimal-time bounded-price way to reach from point A to point B). Both algorithms are performant and have low memory requirements.
```

Comment:
`"Both algorithms are performant and have low memory requirements"`
That's a bold statement, which need to be backed up with either data or runtime / memory complexity e.g. big O notation

Reply:
@swilly22 What is the big O here?

Reply:
@AviAvni correct me if I'm wrong but, both algorithms perform DFS, which make them run in `O(n+m)` `n` being the number of nodes in the graph and `m` number of edges.

Reply:
yes both of them are based on DFS

Reply:
@nermiller saying that both algorithms perform [DFS](https://en.wikipedia.org/wiki/Depth-first_search) entails that their complexity is O(n+m). We can mention both.

Reply:
ok! will update.